Sources/ProjectDescription/TargetAction.swift

```diff
@@ -45,6 +45,9 @@ public struct TargetAction: Codable, Equatable {
     /// Whether to skip running this script in incremental builds, if nothing has changed
     public let basedOnDependencyAnalysis: Bool?
 
+    /// Whether this script only runs on install builds (default is false)
+    public let runOnlyForDeploymentPostprocessing: Bool
```

Comment:
Curious about the naming here, should this match up more closely with what it shows up as in Xcode?

Reply:
it's the same as `PBXShellScriptBuildPhase` uses...

Reply:
but feel free to rename (only keep `codable` in mind)...